src/implementations/react/storybook/package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "build": "build-storybook -c .storybook -o build",
     "start": "start-storybook -p 6006 -s ../../vanilla/packages/ec-preset-website/dist,../../vanilla/packages/ec-preset-editor/dist -c .storybook",
+    "start:eu": "STORYBOOK_SYSTEM=EU start-storybook -p 6007 -s ../../vanilla/packages/ec-preset-website/dist,../../vanilla/packages/ec-preset-editor/dist -c .storybook",
```

Comment:
This I map to "different variants demoed in storybook". I am ok with the approach of having an environment variable for setting an ec/eu context.